runner.yml.dist

```diff
@@ -39,6 +39,27 @@ commands:
     - { task: "run", command: "drupal:settings-setup" }
     - { task: "run", command: "setup:phpunit" }
     - { task: "run", command: "setup:behat" }
+    - task: "append"
+      file: "build/sites/default/settings.override.php"
+      text: |
+        $settings["poetry.identifier.code"] = getenv('POETRY_IDENTIFIER_CODE');
+        $settings["poetry.identifier.sequence"] = getenv('POETRY_IDENTIFIER_SEQUENCE');
+        $settings["poetry.service.username"] = getenv('POETRY_SERVICE_USERNAME');
+        $settings["poetry.service.password"] = getenv('POETRY_SERVICE_PASSWORD');
+        $settings["poetry.notification.username"] = getenv('POETRY_NOTIFICATION_USERNAME');
+        $settings["poetry.notification.password"] = getenv('POETRY_NOTIFICATION_PASSWORD');
+        $config['oe_translation_poetry.settings']['service_wsdl'] = 'http://localhost:8080/build/poetry-mock/wsdl';
+   # Generate settings.testing.php, it will be used when running functional tests.
+    - { task: "process-php", type: "write", config: "drupal.settings", source: "${drupal.root}/sites/default/default.settings.php", destination: "${drupal.root}/sites/default/settings.testing.php", override: true }
+    - task: "append"
+      file: "build/sites/default/settings.testing.php"
+      text: |
+        $settings["poetry.identifier.code"] = getenv('POETRY_IDENTIFIER_CODE');
+        $settings["poetry.identifier.sequence"] = getenv('POETRY_IDENTIFIER_SEQUENCE');
+        $settings["poetry.service.username"] = getenv('POETRY_SERVICE_USERNAME');
+        $settings["poetry.service.password"] = getenv('POETRY_SERVICE_PASSWORD');
+        $settings["poetry.notification.username"] = getenv('POETRY_NOTIFICATION_USERNAME');
+        $settings["poetry.notification.password"] = getenv('POETRY_NOTIFICATION_PASSWORD');
```

Comment:
Why do we not set the mock as the default poetry endpoint in the tests? I know we set it in the setup of the test class but do you foresee at any point using anything but a mock during tests?

Reply:
This to create a .testing.settings file that is used in the test as the default one is not included.